core/app/modules/kit-library/assets/js/pages/index/index.scss

```diff
@@ -6,13 +6,27 @@ $e-kit-library-sticky-search-z-index: 2;
 		overflow-y: auto;
 	}
 
-	&__index-layout-search-area {
+	&__index-layout-top-area {
 		padding: spacing(44);
 		position: sticky;
 		top: -1px;
 		width: 100%;
 		z-index: $e-kit-library-sticky-search-z-index;
 		background-color: var(--app-background-color);
+		gap: spacing( 24 );
+
+		&-search {
+			flex: 1;
+			min-width: 265px;
+		}
+
+		&-sort {
+			min-width: 265px;
+
+			&--safari-support {
+				@include margin-start( spacing( 24 ) );
+			}
+		}
```

Comment:
Maybe combine the `min-width` so when someone in the future will change this property, he won't need to go and search for the second occurrence, both of them will get changed.

Reply:
```CSS
		&-search,
		&-sort {
			min-width: 265px;
		}

		&-search {
			flex: 1;
		}

		&-sort--safari-support {
			@include margin-start( spacing( 24 ) );
		}
```